### PR TITLE
Sawja 1.5.3 release

### PR DESCRIPTION
**Pull request by davidpichardie**

### CODE REVIEW

File: packages/sawja/sawja.1.5.3/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+]
+bug-reports: "sawja@inria.fr"
+author: "Sawja development team"
+install: [
+  [make "install"]
+]
+homepage: "http://sawja.inria.fr"
+available: ocaml-version >= "4.02" 
+depends: [
+  "ocamlfind"
```
Comment (jpdeplaix) [inline packages/sawja/sawja.1.5.3/opam]: Could you add the ```{build}``` attribute here unless you are using the ```Findlib``` library ? This avoids compiling everything everytime ocamlfind gets upgraded/rebuilt.

Comment (davidpichardie) [reply packages/sawja/sawja.1.5.3/opam]: Thanks for your help.
I'm not sure to understand. We are using findlib. 
Which line should I modify? How?

Comment (jpdeplaix) [reply packages/sawja/sawja.1.5.3/opam]: Mmh looking at your code it doesn't seems to be the case. You are using the ocamlfind program but not the ```Findlib``` library (shipped in the same packages).
Unless I'm missing something this line should probably be ```"ocamlfind" {build}``` instead.

For more info you can look at the "Build depends" section in the documentation: https://opam.ocaml.org/doc/Packaging.html#More-on-opam-files

Comment (davidpichardie) [reply packages/sawja/sawja.1.5.3/opam]: Done.

**Comment by davidpichardie** [comment]
Some help will be much appreciated... @jpdeplaix

**Comment by jberdine** [comment]
The only issue I see is the usual alpine being feeble and missing standard things like m4. Unrelated to this PR, right?

**Comment by jpdeplaix** [comment]
@jberdine no, it's mirage-ci being broken not Alpine.

I don't have my laptop with me at the moment but will merge as soon as I can if it's not done already.

**Comment by jberdine** [comment]
Thanks @jpdeplaix! I guess that I have been frustrated enough by alpine in the past that I make unfair assumptions.

**Comment by davidpichardie** [comment]
Thanks a lot @jpdeplaix and @jberdine !

**Comment by jpdeplaix** [comment]
Ok there is another real problem on Alpine this time:
```
#=== ERROR while compiling sawja.1.5.3 ========================================#
# context              2.0.0~rc | linux/x86_64 | ocaml-base-compiler.4.05.0 | file:///home/opam/src
# path                 ~/.opam/4.05.0/.opam-switch/build/sawja.1.5.3
# command              ~/.opam/4.05.0/.opam-switch/build/sawja.1.5.3/./configure.sh
# exit-code            83
# env-file             ~/.opam/log/sawja-16-640243.env
# output-file          ~/.opam/log/sawja-16-640243.out
### output ###
# /home/opam/.opam/4.05.0/.opam-switch/build/sawja.1.5.3/./configure.sh: line 88: fmt: command not found
# /home/opam/.opam/4.05.0/.opam-switch/build/sawja.1.5.3/./configure.sh: line 88: fmt: command not found
# /home/opam/.opam/4.05.0/.opam-switch/build/sawja.1.5.3/./configure.sh: line 88: fmt: command not found
# /home/opam/.opam/4.05.0/.opam-switch/build/sawja.1.5.3/./configure.sh: line 88: fmt: command not found
# 
# /home/opam/.opam/4.05.0/.opam-switch/build/sawja.1.5.3/./configure.sh: line 85: fmt: command not found
```
It's not blocking so that's alright. Maybe you could avoid using this command on the next release or maybe we could add a ```conf-fmt``` meta-package (not sure where it is in Alpine tho)

**Comment by davidpichardie** [comment]
Yes, we will avoid using `fmt` in the next release.

Thanks for your help.

**Pull request merged**